build.gradle

```diff
@@ -81,12 +64,3 @@ task checkstyle(type: Checkstyle) {
 
   classpath = files()
 }
-
-repositories {
-  maven {
-    url 'https://maven.google.com/'
-    name 'Google'
-  }
-}
-
-apply from: 'gradle-mvn-push.gradle'
```

Comment:
if you're deleting this, ya gonna need vanniktech plugin or something else to publish

Reply:
> Old busted Maven publish stuff (I'll fix this in the next PR)

d'oh!